torch/_inductor/optimize_indexing.py

```diff
@@ -63,74 +62,6 @@ def range_expressable_in_32_bits(range):
     )
 
 
-def get_expr_range(expr, vars_ranges: dict):
```

Comment:
Did this get moved?

Reply:
This one was incorrect and it's now replaced by `bound_sympy`.